test/library/packages/ArgumentParser/UsageTest.bad.pt.shadow.good

```diff
@@ -0,0 +1,13 @@
+Config Var Values:
```

Comment:
Hi @arezaii -- I don't think this .good file is used by any tests.  Am I missing it?  (Also, this .good file is identical to `ArgumentParserUsageTest.PtShadow.good`)

Reply:
Hi @cassella, it looks like I moved the test but I didn't remove the .good file. Thanks for catching this.